src/frontend/js/app/entities/reducers.spec.js

```diff
@@ -0,0 +1,129 @@
+import { expect } from 'chai';
+import {
+    createListReducer,
+    initialStateForList,
+    createItemReducer,
+    initialStateForItem,
+    createEntityReducer,
+    initialStateForEntity,
+} from './reducers';
+
+const actionTypes = {
+    item: {
+        REQUEST: 'item/REQUEST',
+        SUCCESS: 'item/SUCCESS',
+        FAILURE: 'item/FAILURE',
+    },
+    list: {
+        REQUEST: 'list/REQUEST',
+        SUCCESS: 'list/SUCCESS',
+        FAILURE: 'list/FAILURE',
+    },
+};
+
+function testListReducer(reducer, initialState) {
```

Comment:
Not mandatory for this PR, but it's better to write the ES6 `const testListReducer = (reducer, initialState)` IMO